src/archive/src/main.rs

```diff
@@ -476,18 +591,47 @@ fn encode_metrics(w: &mut MetricsEncoder<Vec<u8>>) -> std::io::Result<()> {
         stable64_size() as f64,
         "Number of stable memory pages used by this canister.",
     )?;
+    with_call_info(|call_info| {
+        if let Some(successful_fetch) = &call_info.last_successful_fetch {
+            w.encode_gauge(
+                "ii_archive_last_successful_fetch_timestamp_seconds",
+                Duration::from_nanos(successful_fetch.timestamp).as_secs_f64(),
+                "Timestamp of the last successful fetch of entries from the II canister.",
+            )?;
+            w.encode_gauge(
+                "ii_archive_last_successful_fetch_entries_count",
+                successful_fetch.number_of_entries as f64,
+                "Number of entries that were fetched  from the II canister with the last successful fetch.",
+            )?;
+        };
+        Ok::<(), std::io::Error>(())
+    })?;
     Ok(())
 }
 
 /// Publicly exposes the status of the archive canister.
-/// This is useful to check the cycles balance in testing environments.
+/// This is useful to check operations or for debugging purposes.
 #[update]
-async fn status() -> CanisterStatusResponse {
+async fn status() -> ArchiveStatus {
     let canister_id = id();
-    let (status,) = canister_status(CanisterIdRecord { canister_id })
+    let (canister_status,) = canister_status(CanisterIdRecord { canister_id })
         .await
         .expect("failed to retrieve canister status");
-    status
+    let config = with_config(|config| ArchiveInit {
+        ii_canister: config.ii_canister,
+        max_entries_per_call: config.max_entries_per_call,
```

Comment:
what does `max_entries_per_call` correspond to now?

Reply:
`max_entries_per_call` is unchanged still the number of entries _clients_ of the archive can receive per call.

The number of entries fetched is controlled on the II side.